treesapp_abundance.Rmd

```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-Genome assemblies are lossy. Whenever you assemble a (meta)genome from reads, the abundance information is effectively lost, even if it is noted in the FASTA headers by some assemblers. One way to recover these read-level abundances is by aligning the reads back to the contigs and calculating the number of reads that mapped to each. One can go a step further still and use relative abundance normalization measures popular in transcriptomics, such as fragments per kilobase per million mappable reads (FPKM) or transcripts per million (TPM) @Wagner.2012.
```

Comment:
I really did mean lossy - as in data is lost. But I should clearly use a different word :)